doc/docs/user-guide/service/quick-start-guide.md

```diff
@@ -0,0 +1,65 @@
+# Quick Start Guide
+
+---
```

Comment:
According to the template, after the name of the section, we have to write a short sentence (brief) of the page. It will be placed at the center.